debian/rules

```diff
@@ -40,6 +40,14 @@ configure-stamp:
 	./configure --prefix=/usr --localstatedir=/var \
 	  --sysconfdir=/etc $(extraopts) $(confflags) \
 	  $(CEPH_EXTRA_CONFIGURE_ARGS)
+
+	# Wheezy and Precise do not support tracepoints
```

Comment:
you mean "wheezy and precise" because this version of Ceph builds for Wheezy and Precise only? I think we have Trusty and Jessie builds for Hammer too: http://download.ceph.com/debian-hammer/dists/

This change would need to allow those distros to allow this. We do something similar when building:

https://github.com/ceph/ceph-build/blob/master/ceph-build/build/build_deb#L63-L80